ledger/src/blockstore_options.rs

```diff
@@ -215,10 +216,17 @@ pub struct BlockstoreRocksFifoOptions {
 pub const MAX_ROCKS_FIFO_SHRED_STORAGE_SIZE_BYTES: u64 = std::u64::MAX;
 
 impl BlockstoreRocksFifoOptions {
-    fn new(shred_storage_size: u64) -> Self {
-        Self {
-            shred_data_cf_size: shred_storage_size / 2,
-            shred_code_cf_size: shred_storage_size / 2,
+    ///
```

Comment:
Let's complete the function comment describing the behavior when None is specified and maybe describe how it will be used (when the total shred size exceeds the configured number, it will trigger the purge of the oldest file.)

Reply:
Oops, good catch 😄 

I started writing a comment but realized it was going to be fairly similar to the one for `ShredStorageType:: rocks_fifo()`. I just moved some stuff around a little to avoid duplication while still providing good direction